ci/bugfix-release-pr-body.md

```diff
@@ -0,0 +1,63 @@
+AUTO: merge $RELEASE_BRANCH into $SOURCE_BRANCH
```

Comment:
Should this be: "merge $SOURCE_BRANCH into $RELEASE_BRANCH" ?

Reply:
Ha!  Yes.  Yes it should.  Patched in 55c0d0d